src/server.test.ts

```diff
@@ -108,32 +108,32 @@ describe.skip("PUT /signatures/:epoch", () => {
     );
   });
 
-  it("calls findSignatureByEpoch with the given epoch", async () => {
+  it("calls updateSignatureByEpoch with the given epoch and property update", async () => {
     await supertest(app)
       .put("/signatures/1614095562950")
       .send(updateProperties);
-    expect(findSignatureByEpoch).toHaveBeenCalledWith(1614095562950);
+    expect(updateSignatureByEpoch).toHaveBeenCalledWith(1614095562950, updateProperties);
   });
 
   test("when updateSignatureByEpoch returns a signature, it returns a 200 with the given epoch", async () => {
     const response = await supertest(app)
       .put(`/signatures/${passingSignature.epochId}`)
       .send(updateProperties);
-    expect(findSignatureByEpoch).toReturnWith({
+    expect(updateSignatureByEpoch).toReturnWith({
       ...passingSignature,
       ...updateProperties,
     });
     expect(response.status).toBe(200);
     expect(response.body.status).toBe("success");
-    expect(response.body.data).toHaveProperty("signature");
+    expect(response.body.data).toHaveProperty("didUpdate");
```

Comment:
I agree with 11/12 of your changes, but I think I actually prefer the original assertion here.

Why: I think it's useful for a PUT request to receive back the full updated resource (i.e. the full signature) - so that our frontend can then show what the signature _now_ looks like.

(This isn't, however, clear in the test description, and it should be.)

Could you:

1. Rename the test description to make it clearer that we're expecting our response to a successful PUT to include the updated signature?
2. Revert this assertion back to `"signature"` instead of `"didUpdate"`?

Reply:
So, my PUT request _does_  respond back with the full updated resource / the full signature - it just falls under a different key name in this response (vs. the GET response with key "signature"). 

I think these images - which demonstrated example GET and PUT requests I tried out in Postman, where I discovered this - will help shed some light into what I'm trying to say. Esp. in terms of what "didUpdate" is, and why I've changed from "signature" to "didUpdate":

![image](https://user-images.githubusercontent.com/78025318/109842800-5a52e580-7c42-11eb-8c16-270c8d647841.png)

![image](https://user-images.githubusercontent.com/78025318/109842832-60e15d00-7c42-11eb-94c7-6f8b55a3d23d.png)

That having been said, I'm far from sure what "best practice" here is. What do you think?

Reply:
So this is showing that your PUT response has access to the data - which is great!

It's sending it back with the `didUpdate` key, which - as you note - is not so great.

At an educated guess, your response code looks like this:

```js
res.json({
  status: 'success',
  data: {
    didUpdate
  }
})
```

which is shorthand for:
```js
res.json({
  status: 'success',
  data: {
    didUpdate: didUpdate
  }
})
```
using [object property value shorthand](https://alligator.io/js/object-property-shorthand-es6/).

So, what we probably want is:

```js
res.json({
  status: 'success',
  data: {
    signature: didUpdate
  }
})
```

Does that help?

Reply:
It does, thank you so much Richard! Will make those changes to my pull request you suggested now!